Mark as test as expected-failure, OpenSSL is leaking memory.

xfail not skip so we can remove this when OpenSSL doesn't leak anymore.

Upstream-Status: Inappropriate
Signed-off-by: Ross Burton <ross.burton@arm.com>

diff --git a/tests/hazmat/backends/test_openssl_memleak.py b/tests/hazmat/backends/test_openssl_memleak.py
index 835c9256..639f6ba2 100644
--- a/tests/hazmat/backends/test_openssl_memleak.py
+++ b/tests/hazmat/backends/test_openssl_memleak.py
@@ -466,6 +466,7 @@ class TestOpenSSLMemoryLeaks:
             )
         )
 
+    @pytest.mark.xfail(reason="has memory leaks")
     def test_create_certificate_with_extensions(self):
         assert_no_memory_leaks(
             textwrap.dedent(
